app/Http/Controllers/UserController.php

```
@@ -53,6 +53,8 @@ public function store(Request $request)
 
         if ($found_user && password_verify($input['password'], $found_user->password)) {
             $user = $found_user;
+        } else {
+            throw new HttpException(412, 'Incorrect password.');
```

What I'd done in a previous commit was let the `validate` function handle catching the error and throwing the exception instead of it happening here. The error message there specifies if a phone number or email is already taken. I _think_ that's ok, considering this endpoint's meant to be used for user registration. Let me know if you haves cases where we'd want to throw the exception here though.